calico-policies/public-network-isolation/ca-tor/allow-ibm-ports-public.yaml

```diff
@@ -0,0 +1,87 @@
+# This policy opens port 53 and 5353 for DNS,
+# port 2049 for communication with NFS file servers, ports 443 and 3260,
+# for communication to block storage, port 5473 for communication to the
+# calico-typha ClusterIP, port 2040 and 443 for the master
+# API server local proxy, and port 2041 for the etcd local proxy.
+# Port 52311 for Bigfix.
+
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: allow-ibm-ports-public
+spec:
+  egress:
+  - action: Allow
+    destination:
+      ports:
+      - 53
+      - 5353
+      - 443
+      - 2049
+      - 3260
+      - 5473
+    protocol: UDP
+    source: {}
+  - action: Allow
+    destination:
+      ports:
+      - 53
+      - 5353
+      - 443
+      - 2049
+      - 3260
+      - 5473
+    protocol: TCP
+    source: {}
+  - action: Allow
+    destination:
+      ports:
+      - 443
+      nets:
+      - 172.21.0.1/32 # For clusters created <3 years ago
+      - 10.10.10.1/32 # For clusters created >3 years ago
```

Comment:
Since this is a new zone, we don't need the old 10.10.10.1/32 CIDR, this can be removed